### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_Eclipse Marketplace_Keyboard: "Eclipse Marketplace", "Obeo Marketplace" and "Red Hat" are not accessible through keyboard

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3366]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier!

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS2.1.1; #Keyboard; #C+E-H; #Java tooling for IntelliJ/Eclipse; 

**Environment Details:**
Eclipse IDE for Enterprise Java Developers
Version: 2019-06 (4.12.0)
Build id: 20190614-1200

**OS Details:** 
OS: Windows 10 (Version 1903, Build: 18362.175)
MAS Violated: MAS 2.1.1

**Repro Steps: ​**
1) Open Eclipse application
2) Tab till Help and select "Eclipse Marketplace" from the list
3) Search/Find for 'Azure Toolkit for Eclipse' and press enter
4) Check if "Eclipse Marketplace", "Obeo Marketplace" and "Red Hat" are accessible through keyboard

**Actual Result: ​**
"Eclipse Marketplace", "Obeo Marketplace" and "Red Hat" are not accessible through keyboard

**Expected Result: ​**
"Eclipse Marketplace", "Obeo Marketplace" and "Red Hat" should be accessible through keyboard

**User Impact: ​**
Keyboard only users and screen reader users will not be able to access the solutions present under those links
 
**MAS Reference:**
[MAS 2.1.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2})

[Bug_3366_Eclipse_Marketplace_Links_Keyboard.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451856/Bug_3366_Eclipse_Marketplace_Links_Keyboard.pptx)

**Comment by Flanker32** [comment]
@soumyatippapuram This is an external issue which has been linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), please help us verify it, thanks!

**Comment by soumyatippapuram** [comment]
Reported this issue to Eclipse (https://bugs.eclipse.org/bugs/show_bug.cgi?id=551110)

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749768
